### PR TITLE
Add workflows-worker to the docker composition

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

File: testutils/infra/container_manager/docker_compose_manager.py [hunk 1]
```diff
@@ -73,8 +73,8 @@ class DockerComposeNamespace(DockerNamespace):
         COMPOSE_FILES_PATH + "/extra/recaptcha-testing/tenantadm-test-recaptcha-conf.yml",
     ]
 
-    NUM_SERVICES_OPENSOURCE = 13
-    NUM_SERVICES_ENTERPRISE = 17
+    NUM_SERVICES_OPENSOURCE = 14
+    NUM_SERVICES_ENTERPRISE = 18
```
Comment (lluiscampos) [inline testutils/infra/container_manager/docker_compose_manager.py]: Reminder: we'll need a corresponding num services update in `mender-docs` [here](https://github.com/mendersoftware/mender-docs/blob/master/07.Administration/02.Production-installation/docs.md) and [here](https://github.com/mendersoftware/mender-docs/blob/master/07.Administration/02.Production-installation/01.Upgrading-from-OS-to-Enterprise/docs.md).

Comment (lluiscampos) [reply testutils/infra/container_manager/docker_compose_manager.py]: Honestly... Maybe there is room to fix this in a better way?

Once this test in mender-docs has integration repo cloned, it is very simple to grep and extract `NUM_SERVICES_OPENSOURCE` and `NUM_SERVICES_ENTERPRISE` from this file to be used in that test. We could even move it to a separate file more obvious like `infra/defs.py` or something like that (it is not docker-compose specific at all)

**Comment by merlin-northern** [comment]
@tranchitella build servers fails:
```
 + git=workflows
 ++ /builds/Northern.tech/Mender/integration/extra/release_tool.py --map-name docker workflows-worker docker_url
 Don't know how to build docker image workflows-worker
 + docker_url=mendersoftware/workflows-worker
 + case "$docker" in
 + echo 'Don'\''t know how to build docker image workflows-worker'
 + exit 1
```
dont we need something like this in mender-qa:
```diff
diff --git a/scripts/jenkins-yoctobuild-build.sh b/scripts/jenkins-yoctobuild-build.sh
index d445a63..97b908f 100755
--- a/scripts/jenkins-yoctobuild-build.sh
+++ b/scripts/jenkins-yoctobuild-build.sh
@@ -283,7 +283,7 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
         docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)

         case "$docker" in
-            deployments|deployments-enterprise|deviceauth|inventory|tenantadm|useradm|useradm-enterprise|workflows|create-artifact-worker)
+            deployments|deployments-enterprise|deviceauth|inventory|tenantadm|useradm|useradm-enterprise|workflows|workflows-worker|create-artifact-worker)
                 cd go/src/github.com/mendersoftware/$git
                 # Versions before 2.0.0 used "go build", later ones
                 # build everything inside multi-stage docker builds.
```

**Comment by tranchitella** [comment]
@merlin-northern 
https://github.com/mendersoftware/mender-qa/pull/365

**Comment by merlin-northern** [comment]
> @merlin-northern
> [mendersoftware/mender-qa#365](https://github.com/mendersoftware/mender-qa/pull/365)

looks good to me!

**Pull request merged**